examples/3d-tiles/app.js

```diff
@@ -206,30 +215,31 @@ export default class App extends PureComponent {
     );
   }
 
-  _onTileLoaded(tileHeader) {
-    const {name} = this.state;
-    // cannot parse the center from royalExhibitionBuilding dataset
-    const isRoyal = name === 'royalExhibitionBuilding' && tileHeader.depth === 1;
-    if (tileHeader.depth === 0 || isRoyal) {
-      const {center} = tileHeader.boundingVolume;
-      if (!center) {
-        // eslint-disable-next-line
-        console.warn('center was not pre-calculated for the root tile');
-      } else {
-        scratchLongLat.copy(center);
-        if (isRoyal || name === 'TilesetPoints') {
-          Ellipsoid.WGS84.cartesianToCartographic(center, scratchLongLat);
+  _onTilesetLoaded(tileset) {
+    const root = tileset._root;
+    // TODO: won't be needed after data PR merges, just use root
+    const tileHeader = tileset.isRoyal ? root.children[0] : root;
+    const {center} = tileHeader.boundingVolume;
+    if (!center) {
+      // eslint-disable-next-line
+      console.warn('center was not pre-calculated for the root tile');
+    } else {
+      scratchLongLat.copy(center);
```

Comment:
Do we want to include zoom?

Reply:
> Do we want to include zoom? 

I think that ideally yes, we would calculate a web mercator zoom level that covered the bounding volume, as a "suggestion" to the app.

This would allow apps to always show arbitrary tilesets at the appropriate zoom level after load.

But since it some extra work to calculate it, we can add the `zoom` parameter later, or just hard code a "normal" zoom level for now.

Reply:
Gotcha, the normal zoom level is what's there now.